data/help/texts.json

```diff
@@ -220,22 +220,22 @@
     "messages": [
       "You control vehicles using the numpad, or vikeys.  However, you control their controls, rather than the vehicle directly.",
       "<HELP_DRAW_DIRECTIONS>",
-      "In order to assume control of the vehicle, get to a location with working \"vehicle controls\" and a \"seat\", then press <press_control_vehicle>.  Once you are in control, <press_UP> accelerates, <press_DOWN> slows or reverses, & <press_LEFT> & <press_RIGHT> turn left or right.  Diagonals adjust course and speed.  You default to cruise control, so the gas/brake adjust the speed which the vehicle will attempt to maintain.",
+      "In order to assume control of a vehicle, get to a location with working \"vehicle controls\" and a \"seat\", then press <press_control_vehicle>.  Once you are in control, <press_UP> accelerates, <press_DOWN> slows or reverses, & <press_LEFT> & <press_RIGHT> turn left or right.  Diagonals adjust course and speed.  You default to cruise control, so the gas/brake adjust the speed which the vehicle will attempt to maintain.",
       "10-30 MPH, or 16-48 KPH, is a good speed for beginning drivers, who tend to fumble the controls.  As your Driving skill improves, you will fumble less and less.  To simply maintain course and speed, hit <press_pause>.",
       "It's a good idea to pull the handbrake - \"s\" - when parking, just to be safe.  If you want to get out, hit the lights, toggle cruise control, turn the engine on or off, or otherwise use the vehicle controls, press <press_control_vehicle> to bring up the \"Vehicle Controls\" menu, which has options for things you'd do from the driver's seat.",
```

Comment:
No <press_[whatever]> for using the hand brake/emergency brake?

Reply:
I'm not sure what the <press_X> string would be, and I couldn't find it in the rest of the file. There are a couple other places where the string literal is used as well. I'll make another pass and see if I can find out what that might be.

Reply:
Damn, no luck so far. I'll make a note at the bottom of this PR and see if someone can help.